dist/parsed_css.js

```diff
@@ -0,0 +1,646 @@
+//[parsed_css]
+
+try {
+  const MAX_STYLESHEET_BYTES = 500 * 1024; // 500 KB
+  const MAX_AST_BYTES = 4 * 1024 * 1024; // 4 MB
+  const MAX_TOTAL_BYTES = 50 * 1024 * 1024; // 50 MB
+  const stylesheets = $WPT_BODIES.filter(i => i.type == 'Stylesheet').map(i => ({url: i.url, body: i.response_body}));
+  const block = Array.from(document.querySelectorAll('style')).map(i => ({url: 'block', body: i.innerHTML}));
+  const inline = Array.from(document.querySelectorAll('[style]')).map(i => ({url: 'inline', body: i.getAttribute('style')}));
+
+  const parsed_css = stylesheets.concat(block, inline).filter(({url, body}) => {
+    return body.length <= MAX_STYLESHEET_BYTES;
+  }).map(({url, body}) => {
+    return {
+      url,
+      ast: parse(body, {
+        silent: true,
+        inline: url == 'inline'
+      })
+    };
+  }).filter(({url, ast}) => {
+    return JSON.stringify(ast).length <= MAX_AST_BYTES;
+  });
+
+  if (JSON.stringify(parsed_css).length > MAX_TOTAL_BYTES) {
+    return null;
+  }
+
+  return parsed_css;
+} catch (e) {
+  return null;
+}
+
+/* Rework CSS */
+
+// http://www.w3.org/TR/CSS21/grammar.html
+// https://github.com/visionmedia/css-parse/pull/49#issuecomment-30088027
+var commentre = /\/\*[^*]*\*+([^/*][^*]*\*+)*\//g
```

Comment:
Is this french?

```suggestion
const CSS_COMMENT = /\/\*[^*]*\*+([^/*][^*]*\*+)*\//g
```

Reply:
Haha no that's in the original source code. I think they're going for "comment regular expression".

Reply:
Ah ha. Google Translate told me it was French: https://www.google.com/search?q=commentre+in+english

Also `const` doesn't work for RegExes apparently.